### PR TITLE
Add assets + Update README.md

### PR DESCRIPTION
**Pull request by Aayush9029**
#16 Updated example images and README.md accordingly.

Considering adding GIFs/videos to showcase transitions - kept as is for now.

I have opted to use `circular` masks for consistency throughout the project. Please note that some examples previously used `rectangular` masks. If you think it's necessary to modify the example code to maintain uniformity or if you would like to update the corresponding images, please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,49 +16,47 @@ Inferno is an open-source collection of fragment shaders designed for SwiftUI ap
...
-    * [Crosswarp RTL](#crosswarp-rtl)
-    * [Diamond](#diamond)
-    * [Diamond Wave](#diamond-wave)
-    * [Pixellate](#pixellate)
-    * [Radial](#radial)
-    * [Swirl](#swirl)
-* [Inferno Sandbox](#inferno-sandbox)
-* [Contributing](#contributing)
-* [License](#license)
-* [Where to learn more](#where-to-learn-more)
```
Comment (wzso) [inline README.md:60]: These changes are necessary. `*`s function the same as `-`s do.

File: README.md [hunk 2]
```diff
@@ -179,18 +172,18 @@ All the shaders in Inferno were specifically written for readability. Specifical
 3. Have an explanation of the algorithm used.
 4. Provide detailed line-by-line English translations of what the code means.
 
-The combination of what the code *does* (the interlinear comments) and what the code *means* (the algorithm introduction) should hopefully make these shaders comprehensible to everyone.
```
Comment (wzso) [inline README.md:182]: Here, both `*text*` and`_text_` are rendered as _italics_.

**Review by wzso — CHANGES_REQUESTED** [review]
Looks good to me, except for the accidentally replaced asterisks.

**Comment by Aayush9029** [comment]
Those weren't accidental, they're similar to * but more clearer imo.

Especially since ** is a thing.

image

[Source](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)


Willing to change to * though. Need you input

File: README.md (hunk 1 above)
Comment (Aayush9029) [reply README.md:60]: Sorry, I’m slightly confused. Are you simply noting that using ‘-’ is acceptable, or do you want me to change it?

**Comment by Aayush9029** [comment]
Italics: _ is often preferred over * for clarity, as * is also used for bullet points and bold text.
Bullet Points: - is commonly used, distinguishing it from the asterisk's other uses.
Bold Text: ** is the standard, widely recognized for bolding.
Bold and Italics: Both *\*\* and \*\*\_ are used, with the choice based on personal preference.


\*\*HELLO WORLD*\*‎ ‎ ‎ ‎ ‎ ‎‎ ‎  ‎ ‎ =‎ ‎‎ ‎ ‎ ‎  ‎  **HELLO WORLD**

\*\*\*HELLO WORLD\*\*\*‎ ‎ ‎ ‎ ‎ ‎ =‎ ‎ ‎ ‎ ‎ ‎  ***HELLO WORLD***

\*\*\_HELLO WORLD\_\*\*‎ ‎ ‎ ‎ ‎ ‎ =‎‎ ‎ ‎ ‎ ‎   **_HELLO WORLD_**

**Comment by twostraws** [comment]
This is perfect – thank you so much!

**Comment by Aayush9029** [comment]
@twostraws 

> Considering adding GIFs/videos to showcase transitions - kept as is for now.

Do you have any input on this? 

The animations are so pleasing, we must display them at all cost xD.

**Comment by twostraws** [comment]
You're welcome to try, but keep in mind this repo will be checked out by many people now that we're adopting SPM – if we add some big GIFs etc, it's going to be a pretty significant hit. If you can create small, short videos using HEVC/H265, they ought to be tiny!

**Pull request merged**